astropy/table/tests/test_table.py

```diff
@@ -1018,6 +1018,18 @@ def test_rename_by_attr(self, table_types):
         assert np.all(t['c'] == np.array([1, 2, 3]))
         assert np.all(t['a'] == np.array([4, 5, 6]))
 
+    def test_rename_columns(self, table_types):
+        self._setup(table_types)
+        t = table_types.Table([self.a, self.b, self.c])
+        t.rename_columns(('a', 'b', 'c'), ('aa', 'bb', 'cc'))
+        assert t.columns.keys() == ['aa', 'bb', 'cc']
```

Comment:
The idiom for astropy Table is `t.colnames` not `t.columns.keys()`.  Use this in the test here.